src/Mobs/Monster.cpp

```diff
@@ -1129,7 +1135,6 @@ bool cMonster::WouldBurnAt(Vector3d a_Location, cChunk & a_Chunk)
 		(a_Chunk.GetSkyLight(RelX, RelY, RelZ) == 15) &&             // In the daylight
```

Comment:
I fixed it in this particular case, but this could happen ANYWHERE, lots of stuff, including many mob types, inhert the tick() function.